playbooks/roles/shadowsocks/defaults/main.yml

```diff
@@ -4,3 +4,7 @@ shadowsocks_local_port: "1080"
 shadowsocks_timeout: "600"
 shadowsocks_encryption_method: "chacha20-ietf-poly1305"
 shadowsocks_tcp_fast_open: "true"
+shadowsocks_obfs_plugin: "obfs-server"
+shadowsocks_client_obfs_plugin: "obfs-local"
+shadowsocks_obfs_plugin_opts: "obfs=http"
+shadowsocks_obfs_client_plugin_opts: "obfs=http;obfs-host=www.cloudfront.com"
```

Comment:
When you replace `www.cloudfront.com` can you put it into its own var? e.g. `shadowsocks_obfs_cover_domain: "www.github.com"` and then set `shadowsocks_obfs_client_plugin_opts: "obfs=http;obfs-host={{ shadowsocks_obfs_cover_domain }}". That lets us use the var in the documentation instead of copy/pasting.